src/components/SunErrorContent.ts

```diff
@@ -0,0 +1,24 @@
+import { html, TemplateResult } from "lit-html"
```

Comment:
Can we change to single quotes here and in index?

Reply:
Could you perhaps provide some more details? I don't get/understand the issue.

Reply:
Nvm, I got it! Changed all double quote imports I could find to single quote.